mesonbuild/backend/ninjabackend.py

```diff
@@ -1429,7 +1429,7 @@ def generate_cs_compile_rule(self, compiler, outfile):
         rule = 'rule %s_COMPILER\n' % compiler.get_language()
         invoc = ' '.join([ninja_quote(i) for i in compiler.get_exelist()])
         command = ' command = %s $ARGS $in\n' % invoc
-        description = ' description = Compiling cs target $out.\n'
+        description = ' description = Compiling C# target $out.\n'
```

Comment:
Is this not a comment?

Reply:
I don't think there's actually any danger of this being treated as a ninja comment.  It's perhaps not totally clear in the ninja manual, but it seems that `#` starts a comment only when it's the first non-whitespace character on a line. (this is good, as otherwise there would be no way to write a literal `#` )